pkg/e2e/osd/dedicatedadmin.go

```diff
@@ -70,5 +75,135 @@ var _ = ginkgo.Describe(dedicatedAdminTestName, func() {
 			Expect(err).To(HaveOccurred())
 
 		}, float64(viper.GetFloat64(config.Tests.PollingTimeout)))
+
+		// sa can create project
+		ginkgo.It("sa can create project", func() {
+
+			h.SetServiceAccount("system:serviceaccount:%s:dedicated-admin-project")
+
+			_, err := createNamespace("cust-ns", h)
+			Expect(err).NotTo(HaveOccurred())
+
+		}, float64(viper.GetFloat64(config.Tests.PollingTimeout)))
+
+		// regular dedicated-admin user can create admin rolebinding
+		ginkgo.It("can create admin rolebinding", func() {
+
+			daGroup, err := h.User().UserV1().Groups().Get(context.TODO(), "dedicated-admins", metav1.GetOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			daGroup.Users = append(daGroup.Users, "dummy-user@redhat.com")
+			daGroup, err = h.User().UserV1().Groups().Update(context.TODO(), daGroup, metav1.UpdateOptions{})
+			Expect(err).NotTo(HaveOccurred())
+			h.Impersonate(rest.ImpersonationConfig{
+				UserName: "dummy-admin@redhat.com",
+				Groups: []string{
+					"dedicated-admins",
+				},
+			})
+			defer func() {
+				h.Impersonate(rest.ImpersonationConfig{})
+			}()
+
+			_, err = createRolebinding("dummy-user@redhat.com", "admin", h)
+			Expect(err).NotTo(HaveOccurred())
+
+		}, float64(viper.GetFloat64(config.Tests.PollingTimeout)))
+
+		// can create edit rolebinding
+		ginkgo.It("can create edit rolebinding", func() {
```

Comment:
What can create "edit" rolebinding

Reply:
Also, may be good to put edit in single quotes to signify its a name.

Reply:
This is a clusterrole right? So it should b a clusterrolebinding

Reply:
+1 fixed syntax to match your comments.

Reply:
'edit' is a clusterrole but we are creating a rolebinding here because we're limiting the 'edit' permissions to specific customer ns